# Represents a KDL::RotationalInertia instance.

float64[9] data
